### ISSUE TITLE
Remember the titles of each tab

### ISSUE THREAD

**Issue by arteze** [lxde/lxterminal#129]
I made a script that changes the tab title, but if I change the tab and go back to the tab, it changes to the original title that it had before.

I want the titles to work correctly, or to have a command available to me that changes the titles and keeps them different for each tab.

I also want that if I change a title in the script window, it also changes in the tab.

```sh
xdotool getactivewindow set_window --name "$@"
PS1="$PS1\[\e];$@\a\]"
```

I also have the problem that I have to run these commands from a script, but if I run them directly in the terminal, the title set in PS1 if I change the tab and come back, stays static, and then it is impossible to change it.

In short... I want there to be some command so that I can have each title in a window and tab and that they are different even when changing tabs.
